src/components/EventList.tsx

```diff
@@ -36,7 +40,7 @@ const EventList = ({ events, org } : EventListProps) : JSX.Element => {
                         <View data-test="event-title">
                             { e.title ? e.title : e.activity.title }
                         </View>
-                        <View data-test="org-title">{ org?.title }</View>
+                        <View data-test="org-title">{ org!.title }</View>
```

Comment:
We've moved the problem up one level. This line means that we claim to know that `org` is never `undefined`. If we know as much, then why do we accept `undefined` in `EventListProps`?

Either this line is wrong, which I don't think it is, or the props types are wrong.